Document equalizer in Cambridge Audio integration (#43540)
home-assistant/home-assistant.github.io: source/_integrations/cambridge_audio.markdown

- Audio output (Speaker select)
- Control Bus mode
- Room correction
- Equalizer state

### Pre-Amp

When Pre-Amp mode is enabled, Home Assistant can control the output volume of your device.

### Control Bus

> When Control Bus mode is enabled, Home Assistant can control the output volume of your power amplifier when it is connected to a Cambridge Audio network player using the Control Bus interface. In this case Pre-Amp mode can be disabled, the network player will send the signal with full volume to the power amplifier and the volume of the power amplifier can be controlled using volume up and down controls. Control Bus does not support setting the volume to a certain value, it can only increase and decrease the current volume.

Reviewer: ```suggestion
When Control Bus mode is enabled, Home Assistant can control the output volume of your power amplifier when it is connected to a Cambridge Audio network player using the Control Bus interface. In this case, Pre-Amp mode can be disabled. The network player will send the signal at full volume to the power amplifier, and the power amplifier's volume can be controlled using the volume up and down controls. The Control Bus does not support setting the volume to a certain value; it can only increase and decrease the current volume.
```

### The ability to change volume is missing

> Volume control is only supported on all-in-one amplifiers, streamers that are in Pre-Amp mode, or streamers that are connected to an amplifier using the Control Bus interface.

Reviewer: ```suggestion
Volume control is supported only on all-in-one amplifiers, streamers in Pre-Amp mode, or streamers connected to an amplifier via the Control Bus interface.
```
